tools/c7n_azure/c7n_azure/resources/storage.py

```diff
@@ -24,3 +29,50 @@ class resource_type(ArmResourceManager.resource_type):
         client = 'StorageManagementClient'
         enum_spec = ('storage_accounts', 'list', None)
         diagnostic_settings_enabled = False
+
+
+@Storage.action_registry.register('setNetworkRules')
```

Comment:
in terms of casing, set-network-rules would be inline with current action naming standards across providers